portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -16,6 +16,12 @@
 
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
```

Comment:
remember to keep your imports in sorted order

Reply:
I'm not sure what the desired action on this is; I found this [Stack Overflow link about ordered Java imports](https://stackoverflow.com/questions/7735671/order-of-import-statements-in-java#:~:text=import%20statements%20should%20be%20sorted,one%20blank%20line%20between%20groups.&text=The%20import%20statement%20location%20is%20enforced%20by%20the%20Java%20language.), should I be grouping the inputs by spaces? Or is this alphabetical order -- what's the correct sorted order here?

Reply:
This usually means alphabetical order. So in this case the com.google.appengine.api.datastore block should be above the com.google.appengine.api.users rows.

Reply:
yeah I meant alphabetical order, sorry for not being clear!